.github/workflows/default.yaml

```
@@ -122,7 +130,11 @@ jobs:
 
       - name: Run build on Linux
         if: runner.os == 'Linux'
-        run: ./test-builds.sh
+        env:
+          CC: ${{ matrix.compiler.CC }}
+          CXX: ${{ matrix.compiler.CXX }}
+        run: |
+          ./test-builds.sh --cleanup
```

If adding support for clang tests does not require adding this option, let's not add it in this PR:

```suggestion
          ./test-builds.sh
```

Otherwise, please add a YAML comment to explain why removing build logs is a good idea when building with clang.